docs/examples/en/controls/OrbitControls.html

```diff
@@ -87,7 +87,7 @@ <h3>[property:Boolean autoRotate]</h3>
 		<h3>[property:Float autoRotateSpeed]</h3>
 		<p>
 			How fast to rotate around the target if [page:.autoRotate] is true. Default is 2.0, which equates to 30 seconds
-			per rotation at 60fps.<br> Note that if [page:.autoRotate] is enabled, you must call [page:.update]
+			per round at 60fps.<br> Note that if [page:.autoRotate] is enabled, you must call [page:.update]
```

Comment:
English is not my first language so I'm not sure if `round` is an improvement over `rotation`. Wouldn't it be more clear to use the term `per orbit`?

Reply:
![image](https://user-images.githubusercontent.com/19889378/105626658-f895a380-5e6b-11eb-91bc-3e4aad5e710a.png)

This word is the word used in the description of `autoRotateSpeed` ​​in OrbitControls.js. My first language is Chinese. However, I think `round `is better understood than `orbit`.  😄

Reply:
I believe `orbit` is the technical correct term. But if the inline docs already established `round`, I'm fine with it.

Reply:
Yes, `orbit` is best in US English -- or `rotation`. `round` is not common.

Reply:
@mrdoob What's your opinion？

Reply:
I think @WestLangley's suggestion sounds good.

Reply:
OK. I will submit it again later.

Reply:
Done, thank you for your suggestions. @Mugen87 @WestLangley @mrdoob